README.md

```diff
@@ -335,6 +322,27 @@ tokenName: 'token',
 tokenRoot: false,
 
 
+// Refresh Token Options
+// =====================
+
+// Option to turn refresh tokens On/Off
+useRefreshToken: false,
+// The option to enable/disable the automatic refresh of Auth tokens using Refresh Tokens
+autoUpdateToken: true,
+// Oauth Client Id
+clientId: false,
+// The property name used when storing the refresh token locally
+refreshTokenStorage: 'aurelia_refresh_token',
```

Comment:
@doktordirk do you mean to move the storage options into a new group of their own? We can do that in another commit if that's okay?